cached_proc_macro/src/cached.rs

```diff
@@ -202,6 +219,7 @@ pub fn cached(args: TokenStream, input: TokenStream) -> TokenStream {
     let no_cache_fn_ident = Ident::new(&format!("{}_no_cache", &fn_ident), fn_ident.span());
 
     let lock;
+    let lock_key;
```

Comment:
It cannot be lock and lock_key at the same time. So you can use the same value, which can be `lock`.